assets/Gruntfile.js

```diff
@@ -199,7 +199,21 @@ module.exports = function (grunt) {
           'bower_components/messenger/build/css/messenger-theme-air.css',
           'bower_components/messenger/build/css/messenger-theme-ice.css',
           'bower_components/messenger/build/js/messenger-theme-future.js',
-          'bower_components/fontawesome/css/font-awesome.css'
+          'bower_components/fontawesome/css/font-awesome.css',
+          "bower_components/patternfly/dist/js/patternfly.js",
```

Comment:
@spadgett @sg00dwin so we were never including patternfly.js before (it was excluded because we were overriding the main for patternfly in our bower.json), do we want to go ahead and start including it?  looks like it gives us some c3 chart defaults and stuff